### ISSUE TITLE
session files accumulate in syspass/app/cache

### ISSUE THREAD

**Issue by rob42** [nuxsmin/sysPass#1335]
**sysPass Version**
3.0.5

**Describe the bug**
the syspass/app/cache dir is accumulating session files about 4 per minute, and they are never cleared. This eventually fills the disk.

**To Reproduce**
This is a standard install of syspass 3.0.5 on ubuntu 16.04, php 7.0, nginx 1.10.3-0ubuntu0.16.04.3

This `ls -al` below is just a sample, the dir is 3.2Gb, all the same. 
disk shows:
```
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:36 5cac13590bae17.17121695
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:36 5cac1359739602.32214737
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac13658629e0.29970033
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac1369b28285.91909517
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac138388c582.87075301
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac1387b48e20.86380762
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac13950ba814.69283429
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:37 5cac1395734126.26035132
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13a18c89c0.19969587
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13a5b5d451.83964173
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13bf90f8f6.60114320
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13c3b804f1.23619555
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13d109a9e5.51852332
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:38 5cac13d170f395.55602554
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac13dd942140.38966042
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac13e1be3ab8.10771350
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac13fb98cc17.35190349
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac13ffbbdb13.60747519
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac140d07a420.16311947
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:39 5cac140d6e9477.97360047
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac14199bb0f4.96273378
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac141dbdf4e9.12780843
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac14379e4436.81004585
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac143bbfeda0.70809512
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac1449051d62.66510439
-rw-r--r-- 1 www-data www-data     1159 Apr  9 03:40 5cac14496ca469.53466790
```

**Expected behavior**
The cache should clear itself. 

**Screenshots**

**Event log**
php-log:
```
[06-May-2019 21:30:28] NOTICE: [pool syspass] child 2599 exited with code 0 after 14989.174343 seconds from start
[06-May-2019 21:30:28] NOTICE: [pool syspass] child 6348 started
[06-May-2019 21:48:15] NOTICE: [pool syspass] child 2763 exited with code 0 after 15006.029688 seconds from start
[06-May-2019 21:48:15] NOTICE: [pool syspass] child 6547 started
[06-May-2019 22:52:47] NOTICE: [pool syspass] child 3526 exited with code 0 after 15021.311383 seconds from start
[06-May-2019 22:52:47] NOTICE: [pool syspass] child 7276 started
[07-May-2019 02:52:21] NOTICE: [pool syspass] child 7276 exited with code 0 after 14374.550548 seconds from start
```
Syslog shows nothing, syspass logs show nothing unexpected.

**Platform (please complete the following information):**
see above

**Additional context**
This could be a problem of my installation, any ideas how to solve it?

**Comment by nuxsmin** [comment]
Hello, this is the correct behavior, since PHP's execution nature (on-time interpreted language), so there would be needed to schedule a cron job for deleting those files.

Regards

**Comment by nuxsmin** [comment]
Hello, I'm trying to find out "who" is storing those files there and sysPass does not store them without any structured path:

![464](https://user-images.githubusercontent.com/1775774/63386710-022bf980-c3a4-11e9-909c-bd3db72db99b.jpg)

**Comment by vmario89** [comment]
whats the fix on this? having a 4.2 GB directory too. I can clear it up completely once per night for example
